projects/packages/jetpack-mu-wpcom/src/features/wpcom-wpadmin-page-view/wpcom-wpadmin-page-view.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * WPCom Tracking for WP Admin Page Views.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom\Wpcom_Wpadmin_Page_View;
+
+use WPCOMSH_Support_Session_Detect;
+
+/**
+ * Bump a Tracks stat for every wp-admin page view not generated by an Automattician.
+ * This is possibly interesting on its own but will also be used as part of an
+ * initiative to understand how much supply we're offering for live chat demand.
+ * See https://nosaraproject.wordpress.com/2015/03/06/live-chat-stats/
+ */
+function wpcom_nosara_track_admin_page_views() {
+	if ( do_not_track_a11ns() ) {
+		return;
+	}
+
+	$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+	$is_atomic_site = ! $is_simple_site;
+
+	global $current_user, $current_blog, $current_screen;
+
+	if (
+		! $current_user instanceof \WP_User ||
+		! $current_screen instanceof \WP_Screen
+	) {
+		return;
+	}
+
+	$blog_id    = null;
+	$user_types = array();
+
+	if ( $is_simple_site ) {
+		if ( ! $current_blog instanceof \WP_Site ) {
+			return;
+		}
+
+		$is_calypsoify_enabled = isset( $_GET['calypsoify'] ) && 1 === (int) $_GET['calypsoify']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		// Calypsoified WP Admin pages should not count as WP Admin page views, since the user is actually in Calypso.
+		if ( $is_calypsoify_enabled ) {
+			return;
+		}
+
+		$blog_id = $current_blog->blog_id;
+		// @phan-suppress-next-line PhanUndeclaredClassMethod -- This class is available in simple sites.
```

Comment:
Can you update the wpcom stubs instead of suppressing the error? See p81Rsd-1Pn-p2 on how to update the stubs.

Reply:
Thanks, I've addressed that.